include/envoy/config/subscription.h

```diff
@@ -75,18 +75,19 @@ template <class ResourceType> class Subscription {
  * Per subscription stats. @see stats_macros.h
  */
 // clang-format off
-#define ALL_SUBSCRIPTION_STATS(COUNTER) \
+#define ALL_SUBSCRIPTION_STATS(COUNTER, GAUGE) \
```

Comment:
nit: align the spacing of `\`

Reply:
hmm I thought `fix_format` would take care of this.